x/incentives/keeper/distribute.go

```diff
@@ -20,22 +21,28 @@ func (k Keeper) getDistributedCoinsFromGauges(gauges []types.Gauge) sdk.Coins {
 	return coins
 }
 
+// getToDistributeCoinsFromGauges returns coins that have not been distributed yet from the provided gauges
 func (k Keeper) getToDistributeCoinsFromGauges(gauges []types.Gauge) sdk.Coins {
 	// TODO: Consider optimizing this in the future to only require one iteration over all gauges.
+	// TODO: Lets make issue for this, requires changing getGaugesFromIterator logic in gauge.go
 	coins := k.getCoinsFromGauges(gauges)
 	distributed := k.getDistributedCoinsFromGauges(gauges)
 	return coins.Sub(distributed)
 }
 
+// getToDistributeCoinsFromIterator utilizes db.iterator to return a list of gauges.
+// From these gauges, coins that have not yet been distributed are returned
 func (k Keeper) getToDistributeCoinsFromIterator(ctx sdk.Context, iterator db.Iterator) sdk.Coins {
 	return k.getToDistributeCoinsFromGauges(k.getGaugesFromIterator(ctx, iterator))
 }
 
+// getDistributedCoinsFromIterator utilizes db.iterator to return a list of gauges.
```

Comment:
ditto for `iterator` if you decide to apply it
```suggestion
// getDistributedCoinsFromIterator utilizes iterator to return a list of gauges.
```